spec/features/admin/schools/managing_2020_participants_spec.rb

```diff
@@ -5,15 +5,15 @@
 RSpec.feature "Admin managing 2020 participants", js: true, rutabaga: false do
   scenario "Admin views participants and adds a new one" do
     given_there_is_a_school_with_2020_cohort
-    and_i_am_signed_in_as_an_admin
+    and_i_authenticate_as_an_admin
     when_i_visit the_school_2020_cohort_page
-    then_the_page_should_be_accessible
-    and_percy_should_be_sent_a_snapshot_named("Admin NQT+1 listing page")
+    then_the_page_is_accessible
```

Comment:
This should be `and_the_page_should_be_accessible` and/or `then_the_page_should_be_accessible` because they are matchers — they can fail.